### DISCUSSION TITLE
Do you keep your paperless-ngx server up-to-date?

### DISCUSSION THREAD

**Discussion by astubenbord** [astubenbord/paperless-mobile#160] [Polls]
Due to recent changes to the API, I'd like to know how many of you regularly update their paperless-ngx instance? This is of interest because maintaining support for older versions is hard given the increased complexity of maintaining several API versions. So, before I restructure the code to support older versions (which is a lot of effort for me), I'd like to know if this is necessary at all.

If you don't update, I'd also like to know *why*, maybe there are good reasons to keep support for older versions that I just don't know of.

Thanks!

**Comment by ghost** [comment]
I think it is relatable and acceptable if an OS piece of software makes assumptions concerning the corresponding API levels. Whenever there is a malfunction with the app I first check for an update in Play Store and then in the docker repo. As long as you don't get paid for it, I dont see a benefit in supporting old API versions just to support people's lazy a**es. ;-)

**Comment by github-Jan** [comment]
I do update now and then not on regular base, but also not only "when i see something important".
Expectation for a (resonable) new paperless should be fine.
(But also expectation that a API is quite stable)
